src/audio/google/google_ctc_audio_processing.toml

```diff
@@ -0,0 +1,17 @@
+	REM # CTC module config
+	[[module.entry]]
+	name = "CTC"
+	uuid = "BF0E1BBC-DC6A-45FE-BC90-2554CB137AB4"
+	affinity_mask = "0x1"
+	instance_count = "40"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "9"
+	auto_start = "0"
+	sched_caps = [1, 0x00008000]
+	REM # pin = [dir, type, sample rate, size, container, channel-cfg]
+	pin = [0, 0, 0xfeef, 0xf, 0xf, 0x45ff, 1, 0, 0xfeef, 0xf, 0xf, 0x1ff]
+	REM # mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
+	mod_cfg = [0, 0, 0, 0, 4096, 1000000, 128, 128, 0, 0, 0]
```

Comment:
Not really related to this PR, but I'm seeing this pattern abused in the TOML stuff and it bugs me: the TOML entries in the manifest are stored per-type, not per-component.  They shouldn't have stuff that is already covered by the per-component topology data (and especially shouldn't have stuff that duplicates and might conflict with topology).

But people are doing it anyway, because topology doesn't exist in a general form down here in the firmware, where the TOML-generated manifest entries do.  It's a design wart.